src/test/java/test/github2440/TimeoutWithNoExecutorTest.java

```diff
@@ -0,0 +1,27 @@
+package test.github2440;
+
+import org.testng.ITestResult;
+import org.testng.TestListenerAdapter;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import org.testng.xml.XmlClass;
+import org.testng.xml.XmlSuite;
+import org.testng.xml.XmlTest;
+import test.SimpleBaseTest;
+
+import java.util.Collections;
+
+public class TimeoutWithNoExecutorTest extends SimpleBaseTest {
+
+    @Test
+    public void testTimeout() {
```

Comment:
@myemperor - There are no assertions in this test. Can you please help add assertions (using `Assertions`) so that it also validates the expected behavior.

Also please add this test as part of `src/test/resources/testng.xml`

Reply:
done.